charts/karmada/_crds/bases/policy/policy.karmada.io_clusterpropagationpolicies.yaml

```diff
@@ -49,6 +49,19 @@ spec:
                   automatically. e.g. a ConfigMap referred by a Deployment. default
                   false. Deprecated: in favor of PropagateDeps.'
                 type: boolean
+              conflictResolution:
+                default: Abort
+                description: "ConflictResolution declares how potential conflict should
+                  be handled when a resource that is being propagated already exists
+                  in the target cluster. \n It defaults to \"Abort\" which means stop
+                  propagating to avoid unexpected overwrites. The \"Overwrite\" might
+                  be useful when migrating legacy cluster resources to Karmada, in
+                  which case conflict is predictable and can be instructed to Karmada
+                  take over the resource by overwriting."
+                enum:
+                - Abort
+                - Overwrite
```

Comment:
Should the initial letter be capitalized? The previous enum is `overwrite`. If we change to `Overwrite`, will it affect the use of some existing users.

Reply:
Yes, all enumerations should be in `CamelCase`.

> The previous enum is overwrite. If we change to Overwrite, will it affect the use of some existing users?

That's a good point! We might need to take care of it during implementation.